src/lib/list/selection-list.ts

```diff
@@ -479,6 +477,24 @@ export class MatSelectionList extends _MatSelectionListMixinBase implements Focu
     }
   }
 
+  /**
+   * Sets the selected state on all of the options
+   * and emits an event if anything changed.
+   */
+  private _toggleAll(isSelected: boolean) {
```

Comment:
How about `_setAllOptionsSelected`? To me "toggle" means specifically flipping from what it was before.